src/30_Advanced/Using_the_AMP_Validator_API.html

```diff
@@ -0,0 +1,84 @@
+<!---{
+  "hideCode": true,
+  "hidePreview": true,
+  "skipValidation": true
+}--->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/advanced/using_the_amp_validator_api/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+</head>
+<body>
+<!--
+
+## Introduction
+
+The AMP Validator API is an endpoint developers can call to check that their AMP pages are formatted correctly, even before they are deployed. The API can validate live AMP pages as well as local html files.
+
+## See How It Works
+
+To check if a webpage’s AMP markup is correct, just send the AMP page to the endpoint <code>https://amp.cloudflare.com/q/</code> like this:
+
+```
+curl https://amp.cloudflare.com/q/amp.usatoday.com/story/82055560/
+{
+  "source": "http://amp.usatoday.com/story/82055560/",
+  "valid": true,
+  "version": "1488238516283"
+}
```

Comment:
Is it OK if we use https://ampbyexample.com as an example instead of USA Today?

It would be helpful to visually differentiate the command from the output, e.g. by adding `$ ` before the comment, e.g.:

```
$ curl https://amp.cloudflare.com/q/amp.usatoday.com/story/82055560/
{
   "source": "http://amp.usatoday.com/story/82055560/",
   "valid": true,
   "version": "1488238516283"
 }
```
The same applies to the other input samples.